Source/WebKit/UIProcess/BrowsingContextGroup.cpp

```diff
@@ -153,12 +152,18 @@ void BrowsingContextGroup::addPage(WebPageProxy& page)
     });
 }
 
+void BrowsingContextGroup::addRemotePage(WebPageProxy& page, Ref<RemotePageProxy>&& remotePage)
+{
+    auto it = m_remotePages.find(page);
+    if (it == m_remotePages.end())
+        return;
```

Comment:
If we don't have a HashSet here, wouldn't we want to add one?  I'd expect a call to WeakHashMap::ensure or an ASSERT_NOT_REACHED because there's something else ensuring this won't happen.